src/CameraHandler.cpp

```diff
@@ -62,47 +64,55 @@ void CameraHandler::listenerReset(domid_t domId)
 
 void CameraHandler::configToXen(xencamera_config_resp *cfg_resp)
 {
-    v4l2_format fmt = mCamera->formatGet();
+    if (mCamera) {
```

Comment:
This and below: I would rework it as we always memset the response if mCamera is null, so sould you please have a helper for that? Which is called at the start of each affected method, so we exit right at the beginning w/o if(mCamera) switches?

Reply:
Partially done. Partially, because dedicated helper doesn't fit here, because we have four different types of reaction to absence of hardware:
- memset for reference type (sizeof can be used on variable itself)
- return void w/o memset
- return 0 w/o memset
- memset for pointer (sizeof is used on datatype)